files/justfiles/secureblue.just

```diff
@@ -668,3 +668,8 @@ audit-secureblue:
             done
         done
     fi
+
+# Debug pastebin for issue reporting
+debug-info: 
+    #!/usr/bin/bash 
+    fpaste --confirm --private <(echo "=== Rpm-Ostree Status ===" && rpm-ostree status) <(fpaste --sysinfo --printonly) <(echo "=== Flatpaks Installed ===" && flatpak list --columns=application,version,options) <(echo "=== Audit Results ===" && ujust audit-secureblue) <(echo "=== Listing Local Overrides ===" && ujust check-local-overrides) <(echo "=== Recent System Events ===" && journalctl -b -p err..alert --since "1 hour ago") <(echo "=== Failed Services ===" && systemctl list-units --state=failed)
```

Comment:
I tested this locally and I'm not seeing any Rpm-Ostree Status in the output

Reply:
strange, it was working, well 1-2 hour later when i get home i will test it further.